lunatrace/bsl/backend/src/app.ts

```diff
@@ -11,11 +11,30 @@
  * limitations under the License.
  *
  */
+import cors from 'cors';
 import dotenv from 'dotenv';
 dotenv.config();
 import Express from 'express';
 
+import { generatePresignedUrl } from './s3/handler';
+import { findVulnsForPackages } from './vulnerabilities/handler';
+
 const app = Express();
+app.use(cors());
+app.use(Express.json());
+
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
+app.get('/api/upload-sbom', generatePresignedUrl);
+
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
+app.post('/api/package/vulns', findVulnsForPackages);
```

Comment:
Whos calling this endpoint?  Shouldn't we be using our data that has been imported into hasura/postgres?